project-base/src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -15,6 +15,12 @@ services:
   Shopsys\ShopBundle\Model\:
     resource: '../../Model/*/*{Facade,Factory}.php'
 
+  League\Flysystem\MountManager:
+    arguments:
+      -
+        local: '@local_filesystem'
+        basic: '@basic_filesystem'
```

Comment:
if you'll change name of "basic_filesystem", change protocol name too

Reply:
I did not... :)